spotpy/unittests/test_database.py

```diff
@@ -66,6 +68,7 @@ def test_csv_single(self):
                           chains=1, save_sim=True)
 
         csv.save(self.like, self.randompar, self.simulations)
+        csv.save(self.like, self.randompar, self.simulations)
```

Comment:
The extra `csv.save` is so that `csvdata` has more than one entry in the list. (It wouldn't make sense to have a single entry in the CSV file anyway, so this doesn't break anything)